consensus/polybft/bridge.go

```diff
@@ -79,6 +130,12 @@ func (b bridge) PostBlock(req *PostBlockRequest) error {
 		}
 	}
 
+	if b.bridgeEventRelayer != nil {
```

Comment:
This is never nil, since you create a dummy implementation if the node is not relayer.